src/agents/quest_agent.py

```diff
@@ -243,7 +243,6 @@ def evaluate_solution(self, game_state: GameState, context: AgentContext, quest:
             quest_name=quest.name,
             context=context,
         )
-        await_streamed_block(likelihood_block, context)
```

Comment:
Still useful, because there are times when we want the webapp to be able to start streaming the block via the file stream, but the agent needs to wait on the result to do the next stream